File: benchmark/results/java-invocation-inline/run-002/b-baseline/output/com/metaengine/demo/identity/services/TokenService.java
package com.metaengine.demo.identity.services;

import java.util.List;

import com.metaengine.demo.identity.aggregates.User;

/** TokenService for the identity domain. */
public class TokenService {

    /** Create a new User from partial input. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a User by id. Returns null if not found. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Users up to the given limit. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete the User with the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-003/b-baseline/output/metaengine_demo/identity/services/token_service.py
"""TokenService for the identity domain."""
from typing import Optional

from metaengine_demo.identity.aggregates.user import User


class TokenService:
    """TokenService domain service."""

    def create(self, input: Optional[User]) -> User:
        """Issue a token for a user."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[User]:
        """Lookup a tokenized user by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[User]:
        """List users with active tokens."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Revoke a user's token."""
        raise NotImplementedError("not implemented")
